agagd/agagd_core/tables.py

```diff
@@ -109,8 +109,8 @@ def render_chapter_id(self, value):
     class Meta:
         model = Member
         attrs = {"class": "paleblue"}
-        fields = ('full_name', 'state', 'join_date', 'country')
-        sequence = ('full_name', 'chapter_id', 'country', 'state', 'join_date', 'member_id')
+        fields = ('full_name', 'state', 'rating', 'join_date', 'country')
+        sequence = ('full_name', 'rating', 'chapter_id', 'country', 'state', 'join_date', 'member_id')
```

Comment:
The [Member model](https://github.com/usgo/agagd/blob/master/agagd/agagd_core/models.py#L12-L39) and db table does not have a rating column. The rating column is available in either the [Players](https://github.com/usgo/agagd/blob/master/agagd/agagd_core/models.py#L12-L39) or [Rating](https://github.com/usgo/agagd/blob/master/agagd/agagd_core/models.py#L12-L39). However, the [Rating](https://github.com/usgo/agagd/blob/master/agagd/agagd_core/models.py#L12-L39) model does produce duplicate entries.

In order for the rating column to be used here it would need to be looked up from either of those models Rating or Players. The column would also need to be included prior to the Meta class and added to our views fiter and values. Table examples are in [tables.py players__rating line 181](https://github.com/usgo/agagd/blob/master/agagd/agagd_core/tables.py#L181) and [tables.py players__sigma lines 186-188](https://github.com/usgo/agagd/blob/master/agagd/agagd_core/tables.py#L186-L188). Note the double underscore, it allows for lookups from other tables. Views examples are [views.py players__pin_player filter lines 207-208](https://github.com/usgo/agagd/blob/master/agagd/agagd_core/views/core.py#L207-L208) and [views/core.py players__rating lines 225](https://github.com/usgo/agagd/blob/master/agagd/agagd_core/views/core.py#L225).